From e742d5b77ec18926293ec5d101470522f67ee159 Mon Sep 17 00:00:00 2001
From: Jonas Gorski <jogo@openwrt.org>
Date: Thu, 15 Aug 2013 12:10:11 +0200
Subject: [PATCH V2 07/13] MIPS: BMIPS: extend BMIPS3300 to include BMIPS32

Codewise there is no difference between these two, so it does not make
sense to treat them differently. Also chip families having one of these
tend to have the other.

Signed-off-by: Jonas Gorski <jogo@openwrt.org>
---
 arch/mips/Kconfig | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -1457,7 +1457,7 @@ config CPU_BMIPS
 	bool "Broadcom BMIPS"
 	depends on SYS_HAS_CPU_BMIPS
 	select CPU_MIPS32
-	select CPU_BMIPS3300 if SYS_HAS_CPU_BMIPS3300
+	select CPU_BMIPS32_3300 if SYS_HAS_CPU_BMIPS32_3300
 	select CPU_BMIPS4350 if SYS_HAS_CPU_BMIPS4350
 	select CPU_BMIPS4380 if SYS_HAS_CPU_BMIPS4380
 	select CPU_BMIPS5000 if SYS_HAS_CPU_BMIPS5000
@@ -1469,7 +1469,7 @@ config CPU_BMIPS
 	select CPU_SUPPORTS_HIGHMEM
 	select CPU_HAS_PREFETCH
 	help
-	  Support for BMIPS3300/4350/4380 and BMIPS5000 processors.
+	  Support for BMIPS32/3300/4350/4380 and BMIPS5000 processors.
 
 config CPU_XLR
 	bool "Netlogic XLR SoC"
@@ -1550,7 +1550,7 @@ config CPU_LOONGSON1
 	select CPU_SUPPORTS_32BIT_KERNEL
 	select CPU_SUPPORTS_HIGHMEM
 
-config CPU_BMIPS3300
+config CPU_BMIPS32_3300
 	bool
 
 config CPU_BMIPS4350
@@ -1644,7 +1644,7 @@ config SYS_HAS_CPU_CAVIUM_OCTEON
 config SYS_HAS_CPU_BMIPS
 	bool
 
-config SYS_HAS_CPU_BMIPS3300
+config SYS_HAS_CPU_BMIPS32_3300
 	bool
 	select SYS_HAS_CPU_BMIPS
 
